doc/api/tty.md

```diff
@@ -88,6 +88,8 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* Unrealiable event handler execution on all Windows platforms. The Event handler is only executed for changes in the width of the window and the console must be in raw mode.
```

Comment:
1. `Unrealiable` → `Unreliable`
2. `The Event handler` → `The event handler`
3. I would prefer @gibfahn's suggestion: `*Note:*  Terminal window resize events are unreliable on Windows. The event handler is only...` There is no reason not to use a whole sentence here.
4. Please try to limit the line length to 80 chars by breaking it into multiple lines if necessary.